ragna/source_storages/_chroma.py

```diff
@@ -35,20 +35,20 @@ def __init__(self) -> None:
 
     def store(
         self,
-        documents: list[Document],
+        corpus: Corpus,
         *,
         chat_id: uuid.UUID,
         chunk_size: int = 500,
         chunk_overlap: int = 250,
     ) -> None:
         collection = self._client.create_collection(
-            str(chat_id), embedding_function=self._embedding_function
+            corpus.name, embedding_function=self._embedding_function
```

Comment:
I'm not sure about this. I think it's a better UX to have names be human-readable eg the GitHub namespace of `user/repo`. If we were to generate some form of ID inside of `ragna`, then this would mean that actual embeddings stored in the `source_storage` would not be user friendly for other libraries ie user's would have to remember the ID they need.

I would prefer to just provide a useful error message in the case of conflicts, and to recommend to users to adopt a naming convention like `user/table_name` or something similar.

Reply:
That is a good point. In that case should we maybe drop the `name` attribute in favor of `id: str`. The user is still free to use whatever ID they want, e.g. human readable, but the name "id" implies that it should be unique.